plugin-docker/che-plugin-docker-runner/src/main/java/org/eclipse/che/plugin/docker/runner/BaseDockerRunner.java

```diff
@@ -319,7 +319,18 @@ protected ApplicationProcess newApplicationProcess(DeploymentSources toDeploy, R
                         unpackedApplication = unpackArchive(application, workDir, applicationFilename + "_unpack");
                     }
                 }
-                hostConfig.setBinds(new String[]{String.format("%s:%s", unpackedApplication.getAbsolutePath(), applicationBindDir)});
+
+                // On Windows binding directory needs to follow URL convention with first / and no colon :
+                // instead of C:\\Users\\user it needs to be /c/Users/user (note as well the lowercase c at first)
```

Comment:
it would be better to avoid slash symbol escaping in the comments, because it is not obvious why two slashes are used here